src/runtime_src/tools/scripts/service_bundle.sh

```diff
@@ -58,6 +58,12 @@ done
 # Funcs start here
 #############################
 
```

Comment:
this is good, but please add the following script that I attached in the CR's description
```
for g in `find /sys/kernel/iommu_groups/* -maxdepth 0 -type d | sort -V`; do
    echo "IOMMU Group ${g##*/}:"
    for d in $g/devices/*; do
        echo -e "\t$(lspci -nns ${d##*/})"
    done;
done;
```

make this as a function and then dump the info into a separate file
field engineer is using this script.

Reply:
Added changes as per comments.